src/viewer.js

```diff
@@ -929,19 +929,20 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
     },
 
     /**
-     * Set the drawer for this viewer, as a supported string or drawer constructor.
-     * @param {String | OpenSeadragon.DrawerBase} drawerCandidate The type of drawer to try to construct
-     * @param { Boolean } [redrawImmediately] Whether to immediately draw a new frame. Default = true.
+     * Request a drawer for this viewer, as a supported string or drawer constructor.
+     * @param {String | OpenSeadragon.DrawerBase} drawerCandidate The type of drawer to try to construct.
+     * @param { Boolean } [mainDrawer] Whether to use this as the viewer's main drawer. Default = true.
+     * @param { Boolean } [redrawImmediately] Whether to immediately draw a new frame. Only used if mainDrawer = true. Default = true.
      * @param { Object } [drawerOptions] Options for this drawer. If falsey, defaults to viewer.drawerOptions
      * for this viewer type. See {@link OpenSeadragon.Options}.
-     * @returns {Boolean} whether the drawer was created successfully
+     * @returns {Object | Boolean} The drawer that was created, or false if the requestd drawer is not supported
      */
-    setDrawer(drawerCandidate, redrawImmediately = true, drawerOptions = null){
+    requestDrawer(drawerCandidate, mainDrawer = true, redrawImmediately = true, drawerOptions = null){
```

Comment:
With this many parameters, does it make sense to take in an `options` object with named properties?

Reply:
I don't feel strongly one way or the other. I switched to an `options` object in the most recent commit.

Reply:
Cool. I don't really have a good rule for when we should or shouldn't follow that pattern. This looks good though.